src/angular/15-order-service/order.component-solution.ts

```diff
@@ -24,12 +24,12 @@ function minLengthArray(min: number) {
 export class OrderComponent implements OnInit, OnDestroy {
   orderForm: FormGroup;
   restaurant: Restaurant;
-  isLoading: boolean = true;
+  isLoading = true;
   items: FormArray;
-  orderTotal: number = 0.0;
+  orderTotal = 0.0;
   completedOrder: Order;
-  orderComplete: boolean = false;
-  orderProcessing: boolean = false;
+  orderComplete = false;
+  orderProcessing = false;
   private subscription: Subscription;
```

Comment:
I like that we're adopting the `unSubscribe` pattern for handling subscriptions on `ngOnDestroy`. Is it within scope of this refactor to update here as well?